Dockerfile

```diff
@@ -1,4 +1,26 @@
-FROM phusion/baseimage:jammy-1.0.1
+FROM ubuntu:jammy as builder
+
+WORKDIR /root
+
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \
+        git ca-certificates
+
+RUN git clone --branch master --single-branch https://github.com/phusion/baseimage-docker.git && \
```

Comment:
This could make sense, but I'd rather checkout a specific commit hash in case the script changes (incompatibly at best, with potential security implications at worst).

Incidentally, I'd probably have added something like:
```
USER root
RUN apt-get update && \
    apt-get upgrade -y && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```
Though granted, a year later that might add up to much the same as doing it from scratch. But on the plus side it shouldn't take any special effort in the future.

Reply:
Phusion's scripts already call `apt-get update` and `apt-get dist-upgrade`:

https://github.com/phusion/baseimage-docker/blob/master/image/prepare.sh#L17

https://github.com/phusion/baseimage-docker/blob/master/image/prepare.sh#L41

So any packages that are out of date in the Ubuntu base images will get updated when the image is built.